docs/CSBR.md

```diff
@@ -590,32 +950,91 @@ Because some Application Software Suppliers utilize non-standard revocation mech
 
 ### 4.9.12 Special requirements re key compromise
 
+See [Section 4.9.1](#491-circumstances-for-revocation).
+
 ### 4.9.13 Circumstances for suspension
 
+[EDITOR: This is a controversial issue because some CAs might allow suspension of Code Signing Certificates because the scope is different. Suggest adding an effective date if we are to include this requirement and subsequent statements in 4.9.14, 4.9.15, 4.9.16.]
+
+The Repository MUST NOT include entries that indicate that a Certificate is suspended.
+
 ### 4.9.14 Who can request suspension
 
+Not applicable.
+
 ### 4.9.15 Procedure for suspension request
 
+Not applicable.
+
 ### 4.9.16 Limits on suspension period
 
+Not applicable.
+
 ## 4.10 Certificate status services
 
 ### 4.10.1 Operational characteristics
 
+[EDITOR: We might need to keep these entries even after the expiration of Code Signing or Timestamping Certificates]

```

Comment:
Add references to the more appropriate CSBR section or "fix" the "MAY"

Reply:
The requirement for 10y after expiration should be addressed in a separate ballot.